packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -2217,6 +2217,7 @@ export default class Meeting extends StatelessWebexPlugin {
                   memberId: contentShare.beneficiaryId,
                   url: contentShare.url,
                   shareInstanceId: contentShare.shareInstanceId,
+                  annotationInfo: contentShare.annotation,
```

Comment:
can you also update the jsdoc for the "meeting:startedSharingRemote" event in lines 291-299 and add annotationInfo there? It looks like those comments are very outdated. Can you fix the other properties there too, please?

Reply:
Added it.